### PR TITLE
New script that will revert the stable tag for Jetpack in wp.org svn. 

### PR DESCRIPTION
**Pull request by dereksmart**
#### Changes proposed in this Pull Request:

This adds a new script that does one thing only, which is to roll back the stable tag of the Jetpack plugin in the wp.org repo. It is only meant to be used in case of extreme emergency situations. 

We have discussed in depth internally about this. 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Internal janitorial feature I guess? 

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

**PLEASE BE CAREFUL WHEN TESTING**
There are a couple of confirmations along the way, and I suspect you still will need to input your svn creds in order for the commit to go through. But I have not dared to go beyond the last confirmation myself. 

I would like feedback on the following: 
- Whether we need more or less confirmations during the script
- The verbiage used
- If we should add more context in the description/text on when to use this

#### Proposed changelog entry for your changes:

N/A

### CODE REVIEW

File: tools/revert-release.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NOCOLOR='\033[0m'
+
+# Current stable version
+CURRENT_STABLE_VERSION=$(curl -s http://api.wordpress.org/plugins/info/1.0/jetpack.json | jq -r .version)
+
+# Get all versions, strip anything with alpha characters such as -beta or trunk.
+LAST_STABLE_TAG=$(curl -s http://api.wordpress.org/plugins/info/1.0/jetpack.json | jq -r '.versions | delpaths([paths | select(.[] | test("[A-Za-z]+"; "i"))]) | keys[-2]' )
```
Comment (dereksmart) [inline tools/revert-release.sh:27]: Is there a better/simpler way to get the most recent tag this that doesn't use a third party lib (`jq`)?

Comment (jeherve) [reply tools/revert-release.sh:27]: `jq` seems to be the easiest way to parse json from what I can tell. I'd be okay requiring it in these cases.

File: tools/revert-release.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# Check for requirements. Using `jq` to easily parse json. Is there a simpler way to get the latest tags?
+if [ -z $(command -v jq) ]; then
+	echo "This script requires the jq library."
+	read -p "Do you want to install it on your system with Homebrew? [y/N]" -n 1 -r
+	if [[ $REPLY != "y" && $REPLY != "Y" ]]; then
+		exit 1;
+	else
+		brew install jq 2>/dev/null
+		echo "Done!"
```
Comment (jeherve) [inline tools/revert-release.sh:15]: I would exit after this, so the person can start from scratch once they've installed jq.

Comment (jeherve) [reply tools/revert-release.sh]: Done in 8fb5f4a0efdbefd0e8d47846a511d2d6383d9653

**Comment by jeherve** [comment]
> Could you also add this new file to .svnignore?

Looks like the whole `tools` directory was already in there. 🥳

File: tools/revert-release.sh (hunk 1 above)
Comment (dereksmart) [reply tools/revert-release.sh:28]: Yeah it's the best I found too. It's just horribly un-readable code.

**Pull request merged**